Makefile

```diff
@@ -52,5 +52,6 @@ clean:
 .PHONY: dev
 dev:
 	go get github.com/pilu/fresh
-	docker-compose up
+	docker-compose start
```

Comment:
`docker-compose` starts _existing_ containers but I don't see any containers being created in the `Makefile`. Is there a real need to start with existing containers and not with _fresh_ ones?